packages/codemods/src/transforms/transform-grid-api-methods-v31-3/__fixtures__/scenarios/hello-world/input.js

```diff
@@ -8,4 +8,4 @@ const gridApi = createGrid(document.body, {
 const col = 'foo';
 const row = {};
 gridApi.getValue(col, row);
-gridApi.getValue('colId', row);
\ No newline at end of file
+gridApi?.getValue('colId', row);
```

Comment:
We should cover each framework with a test case to make sure it works as expected. i.e vue, react, angular as well as the ts and js tests you have here.

Reply:
yes, cleanup and tests are something I am going to work on next